deploy/charts/cert-manager/values.schema.json

```diff
@@ -1185,9 +1185,9 @@
       "type": "string"
     },
     "helm-values.prometheus.servicemonitor.targetPort": {
-      "default": 9402,
+      "default": "http-metrics",
       "description": "The target port to set on the ServiceMonitor. This must match the port that the cert-manager controller is listening on for metrics.",
-      "type": "number"
+      "type": "string"
```

Comment:
Is this a breaking change, or will a number value like `9402` still be accepted by Helm?

Reply:
good call, fixed.